server.js

```diff
@@ -1633,7 +1633,100 @@ app.get('/adminAnalytics', function(req,res) {
     });
 });
 
+app.post('/submitFeedback', function(req, res){
+    if (!req.session.user) {
+        return res.redirect('/');
+    }
+
+    if (req.session.user.type === common.userTypes.ADMIN) {
```

Comment:
what does everyone think about this? are we gunna deny admins permission to submit feedback?

Reply:
This is course feedback only. So yeah I don't see why an admin should be submitting course feedback lol. It's their own course. Unless they're bipolar 😂

Reply:
Agree with Mo, doesn't make sense for a prof to submit feedback for their own course 😆